File: lit-udr/lit-udr-BE/lit-udr/EntityFramework/Model/NewUserData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace lit_udr.EntityFramework.Model
{
    public class NewUserData
    {
        public int Id { get; set; }
        public string Hash { get; set; }
        public bool Writer { get; set; }
        public string NewUserEmmail { get; set; }
        public string processDefinitionId { get; set; }
        public string processInstanceId { get; set; }

    }
}

File: lit-udr/lit-udr-app/src/model/NewUserData.js
export default class NewUserData {
    constructor(Hash,TaskId,ProcessDefinitionId,ProcessInstanceId,Id,NewUserEmail,SimulateFail){
        this.hash= Hash;
        this.taskId = TaskId;
        this.processDefinitionId = ProcessDefinitionId;
        this.processInstanceId = ProcessInstanceId
        this.id = Id;
        this.newUserEmail= NewUserEmail;
        this.simulateFail = SimulateFail;

    }
  }